src/ASM/LR/ASMu2DC1.C

```diff
@@ -18,7 +18,8 @@
 
 bool ASMu2DC1::generateFEMTopology ()
 {
-  if (!(geomB = this->createLRfromTensor()))
+    geomB = std::shared_ptr<LR::LRSpline>(this->createLRfromTensor());
+    if (!geomB)
```

Comment:
Still wrong indentation in L21,22..
And since the return type of `createLRfromTensor()` is already changed to a `shared_ptr`, do you need to pass it through its copy constructor again?

Reply:
fixed, and no. and squashed.